Voycar.Api.Web/Features/Members/Endpoints/Get/Personal/Endpoint.cs

```diff
@@ -0,0 +1,44 @@
+namespace Voycar.Api.Web.Features.Members.Endpoints.Get.Personal;
+
+using Repository;
+using Users.Repository;
+
+
+public class Endpoint : Endpoint<Request, Results<Ok<Response>, BadRequest<NotFound>>, Mapper>
```

Comment:
```suggestion
public class Endpoint : Endpoint<Request, Results<Ok<Response>, BadRequest>, Mapper>
```
Ich glaube das sollte so sein, bitte ausprobieren